generators/svelte/files.js

```diff
@@ -76,14 +77,14 @@ const svelteFiles = {
 	],
 	swagger: [
 		{
-			condition: generator => generator.blueprintConfig.swaggerUi,
+			condition: generator => generator.swaggerUi,
 			path: FRONTEND_SRC_DIR,
 			templates: ['swagger-ui/index.html'],
 		},
 	],
 	jest: [
 		{
-			condition: generator => generator.blueprintConfig.jest,
+			condition: generator => generator.jestTest,
```

Comment:
At main generator we avoid using like this.
Once vitest is added you need 2 independent variables at templates and at config.
Makes more difficult to implement prompt choices. And less clear that this is a choices instead of a feature choice.
its your choice anyway.

Reply:
I intend to use `jest` as an option over here. Due to limitation of JDL overrides at blueprint, I don't see any other viable approach. For now, please respect conventions used under this blueprint.